### PR TITLE
[ADD] dead_mans_switch_therp

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by hbrunn** [comment]
@lfreeke @NL66278 @gfcapalbo ping

**Comment by lfreeke** [comment]
:+1: (test)

File: dead_mans_switch_therp/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Dead man's switch for Therp",
+    "version": "1.0.0",
```
Comment (NL66278) [inline dead_mans_switch_therp/__openerp__.py]: Please put 7.0 in front. Maybe take the opportunity to switch to the shortened headers too?

**Comment by hbrunn** [comment]
@NL66278 done

**Comment by hbrunn** [comment]
ping

**Comment by hbrunn** [comment]
ping

**Pull request merged**